### PR TITLE
Fixed installation error.

### PR DESCRIPTION
**Pull request by kvttt**
Fixed also: https://github.com/OpenNFT/OpenNFT/issues/160

### CODE REVIEW

**Review by da0c — COMMENTED** [review]
numpy >=1.13.1+mkl - 
is a good option to improve the stability of Cholecky decomposition necessary for iGLM

**Review by da0c — COMMENTED** [review]
opencv-python==4.1.0.25 ; platform_system != 'Windows' - 
it looks strange, why we used such a strict OpenCV version for non-Windows platforms?
Who knows? @espdev?

File: setup.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 import subprocess
 
 from distutils.spawn import find_executable
-from distutils.core import DistutilsOptionError
+from distutils.errors import DistutilsOptionError
```
Comment (espdev) [inline setup.py:10]: distutils is deprecated package and will be removed from stdlib of future Python versions. We need to replace it with setuptools.

File: requirements.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 PyQt5>=5.12.0
-numpy>=1.13.1,>=1.13.1+mkl
```
Comment (espdev) [inline requirements.txt:2]: What were the issues with numpy+MKL?

**Comment by espdev** [comment]
@da0c 
> Who knows? @espdev?

No, I don't know. :)

**Comment by ykoush** [comment]
fixed in #166

**Pull request closed without merging**